pkg/sqlmigrations/migrations.go

```diff
@@ -319,6 +319,12 @@ var backwardCompatibleMigrations = []migrationDescriptor{
 		name:   "add CREATEROLE privilege to admin/root",
 		workFn: addCreateRoleToAdminAndRoot,
 	},
+	{
+		// Introduced in v20.2.
+		name:                "add created_by columns to system.jobs",
+		workFn:              alterSystemJobsAddCreatedByColumns,
+		includedInBootstrap: clusterversion.VersionByKey(clusterversion.VersionStart20_2),
```

Comment:
I'm not sure how this works but do we need to mint a new version to put here to ensure existing data dirs / clusters from `master` binaries prior to this patch, but from after VersionStart20_2, i.e. the last couple months, will still run the migration?

Reply:
I think we should. Added.